element-android/pipeline.yml

```diff
@@ -130,6 +130,12 @@ steps:
     command:
       - "diff ./vector/src/main/res/values-id/strings.xml ./vector/src/main/res/values-in/strings.xml"
 
+  # Same thing for Hebrew
+  # If this step fails, it means that Weblate has updated the file in value-he/ so to fix it, copy the file to values-iw/
+  - label: "Hebrew"
+    command:
```

Comment:
it might be better to merge these steps together, so that we don't need to wait for two build agents to start and check out the code?

```suggestion
```

(you'll need to update the `label` as well)

Reply:
Ok, I will merge those 2 steps.
The (small)  problem is that if the first diff fails, we will not be able to see if the second one will fail also, because it will not be executed.